ocw_oer_export/__init__.py

```diff
@@ -1,9 +1,8 @@
-__all__ = ["create_json", "create_csv", "main"]
+__all__ = ["create_json", "create_csv"]
 
 import logging
 
 from .create_csv import create_csv
 from .create_json import create_json
-from .cli import main
```

Comment:
To remove the warning:
`<frozen runpy>:128: RuntimeWarning: 'ocw_oer_export.cli' found in sys.modules after import of package 'ocw_oer_export', but prior to execution of 'ocw_oer_export.cli'; this may result in unpredictable behaviour`
I think this was also slowing down the program execution